src/client/game.cpp

```diff
@@ -1461,7 +1461,7 @@ void Game::copyServerClientCache()
 {
 	// It would be possible to let the client directly read the media files
 	// from where the server knows they are. But aside from being more complicated
-	// it would also *not* fill the media cache and cause slower joining of 
+	// it would also *not* fill the media cache and cause slower joining of
```

Comment:
it would be good to configure your editor to not whitespace-fix unrelated lines

Reply:
I don't think my editor has this feature, maybe I should use git hooks for this.

I readded the whitespace, though I don't think it would hurt to remove it.